Libraries/LibJS/Tests/functions/function-new-target.js

```diff
@@ -0,0 +1,26 @@
+test("basic functionality", () => {
+    function foo() {
+        return new.target;
+    }
+    expect(foo()).toBeUndefined();
+    expect(new foo()).toEqual(foo);
+
+    function bar() {
+        const baz = () => new.target;
+        return baz();
+    }
+    expect(bar()).toBeUndefined();
+    expect(new bar()).toEqual(bar);
+
+    class baz {
+        constructor() {
+            this.newTarget = new.target;
+        }
+    }
+    expect(new baz().newTarget).toEqual(baz);
+});
+
+// FIXME: This does not work as expected as toEval() places the code inside a function :|
+// test("syntax error outside of function", () => {
+//     expect("new.target").not.toEval();
+// });
```

Comment:
does this violate the no commented-out code rule?! :D

Reply:
Shouldn't this be a skipped test rather than a comment tho?

Reply:
hah good point. fixed & resolves the question :)